doc/user_doc/vic_app_dev/using_volumes_with_vic.md

```diff
@@ -0,0 +1,75 @@
+# Using Volumes with vSphere Integrated Containers #
+
+vSphere Integrated Containers supports the use of container volumes. When you create or the vSphere Administrator creates a virtual container host, you or the Administrator specify the datastore to use to store container volumes in the `vic-machine create --volume-store` option. For information about how to use the `vic-machine create --volume-store` option, see the section on `volume-store` in [Virtual Container Host Deployment Options](../vic_installation/vch_installer_options.html#volume-store) in *vSphere Integrated Containers Installation and Configuration*.   
+
+## Obtain the List of Available Volume Stores ##
+
+To obtain the list of volume stores that are available on a virtual container host, run `docker info`.
+
+<pre>docker -H <i>virtual_container_host_address</i>:2376 --tls info</pre>
+
+The list of available volume stores for this virtual container host appears in the `docker info` output under `VolumeStores`.
+
+<pre>[...]
+Storage Driver: vSphere Integrated Containers Backend Engine
+VolumeStores: <i>volume_store_1</i> <i>volume_store_2</i> ... <i>volume_store_n</i>
+vSphere Integrated Containers Backend Engine: RUNNING
+[...]</pre>
+
+## Create a Volume in a Volume Store ##
```

Comment:
There should also be a mention on the `create -v <source to mount anonymous volume` path to adding volume's to the volume store for this section. e.g. `create -v /var/myApp/data busybox` this call will actually create a volume in the `default` store if it is specified. The resulting volume will show up in a subsequent `volume ls`.